### ISSUE TITLE
[Feature] don't show error on a desactivated feed

### ISSUE THREAD

**Issue by geogiraux** [FreshRSS/FreshRSS#7097]
**Is your feature request related to a problem? Please describe.**
I’m always frustrated when I see feeds I have desactivated with the error icon, since the error was my initial reason for me to desactivate it (instead of removing it since I want to keep articles). For instance, when a (url) source is removed, its status changes to error, so I'm desactivating it to "acknowledge" this error, but the error icon remains

**Describe the solution you’d like**
on the feeds column (on the left) and on the "manage feeds" page, displaying the desactivtaed icon instead of the error icon

**Describe alternatives you’ve considered**
none, I can live with this, but I like to keep things clean and I don't want to remind which feeds with errors are "normal" / already managed with new/not managed errors

**Comment by Alkarex** [comment]
What FreshRSS version? Please join a screenshot
Related https://github.com/FreshRSS/FreshRSS/pull/6114

**Comment by geogiraux** [comment]
I'm using 1.24.3, and sorry for missing 6114 (I searched the issues list with "desactivate") which looks quite the same
and looking more closely, the left panel behaviour looks right/correct, my request is only for the manage feeds page : there is no icon but the desactivated feeds are displayed and titles are formatted in bold & red :
![image](https://github.com/user-attachments/assets/eea9950b-f163-44ab-9c16-1036a65e72fa)
![image](https://github.com/user-attachments/assets/3d616a1b-4975-48a2-8caa-7993c9077d98)

**Comment by math-GH** [comment]
could you test with the current `edge` (1.25.0-dev)? As far as I understood it is already fixed there.

Improved with PR #6816

**Comment by Alkarex** [comment]
With edge: 
![image](https://github.com/user-attachments/assets/6b3ed0ef-d14b-4080-bf9c-cb4d8ee15adf)

But in that view, I am not sure I like the idea of not showing any difference between muted feeds that are in error or not.

**Comment by Frenzie** [comment]
> the error was my initial reason for me to desactivate it (instead of removing it since I want to keep articles). 

This is but one of many reasons one might opt to mute a feed. The manage feed page should give the tools to manage. This is a matter for custom CSS imho.

**Comment by geogiraux** [comment]
> could you test with the current `edge` (1.25.0-dev)? As far as I understood it is already fixed there.
> 
> Improved with PR #6816

Yes, fixed in 1.25.0 (as mentionned by Alkarex). I understand the idea of making differences between feeds, but then, what about "changing" the button behavior to show only actives feeds with error ? since I guess this is mostly what one wants to see when using this button : access feeds with error if these feeds are not already "managed". And maybe add a button to show only muted feeds ? 
I agree with Frenzie about the manage feed page, which should ideally provide filters for several attributes of feeds (for instance, type of feeds, feeds with proxy, muted/unmuted), but it seems quite a lot of work if this is managed with custom CSS...
